Boss/Mod/ChannelCreator/Manager.cpp

```diff
@@ -68,7 +71,55 @@ Ev::Io<void> report_proposals( S::Bus& bus, char const* prefix
 namespace Boss { namespace Mod { namespace ChannelCreator {
 
 void Manager::start() {
-	bus.subscribe<Msg::Init
+    min_amount = default_min_amount;
+    max_amount = default_max_amount;
+
+    bus.subscribe<Msg::Manifestation
+            >([this](Msg::Manifestation const& _) {
+        return bus.raise(Msg::ManifestOption{
+                "clboss-min-channel-size"
+                , Msg::OptionType_Int
+                , Json::Out::direct(min_amount.to_sat())
+                , "Minimum channel size CLBOSS will create. "
+                  "Set to an integer number of satoshis. "
+                  "default=500000"
```

Comment:
`ManifestOption` accepts a `std::string` in this position, so it should be possible to get this from the `default_max_amount` value with some amount of `+` and `std::string` concatenation.